src/rust/dbv2/README.md

```diff
@@ -0,0 +1,28 @@
+# Database
+
+This repository provides migrations to create the Econia event database, and Rust types to work with the event database.
+
+The aggregator furthermore adds tables to this database under the `aggregator` schema. Those are disucssed in the `src/rust/aggregator` crate.
+
+# Contributing
+
+When creating a new migration, before committing, please make sure to use [sql-formatter](https://github.com/sql-formatter-org/sql-formatter) with the provided configuration.
+
+Also make sure to add the corresponding documentation to `models.md`.
+
+# Documentation
+
+## diesel
+
+We use [`diesel`](https://crates.io/crates/diesel) to create and run migrations and models.
+
+This crates provides the models for every table in the database.
+
+We also use `diesel` to run migrations. In our docker compose configuration, we have a dedicated docker container that runs `diesel migration run`.
+
+## Notifications
+
+We use [Postgres notifications](https://www.postgresql.org/docs/15/sql-notify.html) to create a real time notification feed for Econia's events.
+
+Each event table has a [trigger](https://www.postgresql.org/docs/15/sql-createtrigger.html) that emits a notification with the name of the event as the channel,
```

Comment:
The docker compose uses Postgres 14 per the aptos docs, but these links are for postgres 15

Will this be an issue?